src/plugins/condition/criterion/TelemetryCriterion.js

```diff
@@ -0,0 +1,83 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2019, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import * as EventEmitter from 'eventemitter3';
+
+export default class TelemetryCriterion extends EventEmitter {
+
+    constructor(telemetryDomainObject, openmct) {
+        super();
+
+        this.telemetryObject = telemetryDomainObject;
+        this.openmct = openmct;
+        this.telemetryAPI = this.openmct.telemetry;
```

Comment:
What is declaring a new ref instead of just this.openmct.telemetry?

Reply:
Readability and it's better to access this.telemetryAPI rather than this.openmct.telemetry every time.

Reply:
Why not just call it this.telemetry? The use of API in the name seems unnecessary.

Reply:
It's easier for me to read when I add the API at the end. this.telemetry could be construed as a telemetry object and be confusing.

Reply:
Fair enough. Why not calling the API directly in line 36 similar to this.openmct.objects in line 37?